src/Client.php

```diff
@@ -391,6 +392,12 @@ public function startSession(array $options = [])
      */
     public function watch(array $pipeline = [], array $options = [])
     {
+        if (is_builder_pipeline($pipeline)) {
```

Comment:
This should allow us to change the typehint to `array|Pipeline` in 2.0, is that correct? Do you know of a way to prepare users that are extending this class for the change?

Reply:
I don't think that's necessary - widening the parameter type in 2.0 is good enough.